src/server.py

```diff
@@ -62,13 +73,43 @@ def response_error(code, phrase):
 def parse_request(client_request):
     """Parse info from a request.  Raise Errors is bad request."""
     http_request = Request(client_request)
-
+    # print(http_request.protocol, http_request.http_method, http_request.headers)
     if http_request.http_method != 'GET':
         raise TypeError("This server only accomodates get requests.")
-    if http_request.correct_get_request():
+    elif http_request.protocol != 'HTTP/1.1':
+        raise ValueError("Incorrect Protocol")
+    elif not http_request.has_host:
+        raise ValueError("No Host found.")
+    else:
         return http_request.uri
+
+
+def resolve_uri(uri):
+    # from pdb import set_trace; set_trace()
```

Comment:
Need a docstring here, and definitely make sure to purge and commented out PDBs.